### DISCUSSION TITLE
Embed Jupyter Outputs in revealJS presentation

### DISCUSSION THREAD

**Discussion by jensbri** [quarto-dev/quarto-cli#6894] [Q&A]
### Description

Hello.

I am working on a revealJS presentation in Quarto 
Instead of creating Python code blocks all over the place, I'd like to embed them from a Jupyter Notebook. 
Unfortunately, that doesn't work for me (following [this guide](https://quarto.org/docs/authoring/notebook-embed.html)).

```
While the latest Jupyter Notebooks
Add support for embedding cell outputs in quarto documents using {{< embed >}}. You can address cells by Id, Tag, or label
```

Now I am wondering if that is due to the different releases of 
- let me call it "Pure Quarto" Current Release is [v1.3.450](https://quarto.org/docs/download/)).
- the VSCode Plugin v1.98.0

I add a **tag** to the external Jupyter code cell (a cell containing one Python Plotly graph at a time) and then reference the ipynb and the said **tag**

Am I doing something wrong or is the VSC Quarto Plugin just not incorporating the lasted addition? 

I get the following output on my Quarto RevealJS Presentation:
`Unable to display output for mime type(s): application/vnd.plotly.v1+json`

Cheers,
Jens

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by jensbri** [reply]
Sure. 

Please check https://github.com/jensbri/QuartoTest 

Thanks

**Comment by cderv** [comment]
@dragonstyle is there a limitation with `embed` and some formats ?

**Reply by dragonstyle** [reply]
There is not - it should work just fine and in local testing this did work just fine:

````
---
Title: Example
format: revealjs
---

## Slide 1

```{r}
plot(cars)
```

## Embed

{{< embed hello.ipynb#fig-polar >}}

## Plotly

{{< embed hello.ipynb#fig-plotly >}}
````

I've attached the zip in case that is helpful!

[Example.zip](https://github.com/quarto-dev/quarto-cli/files/12701229/Example.zip)

**Reply by cderv** [reply]
@jensbri I let you look into that and share more information otherwise. Your repo link does not work BTW

**Reply by jensbri** [reply]
Kia ora.

I think these two things fixed my error (the embeddedJupyter Notebook didn't render).
1. `pio.renderers.default = "notebook"` was added to some relevant blocks of the notebook
2. I added this to my yaml header of the qmd file

```
execute: 
  enabled: true
```

Thanks for your help
